pkg/compiler/compiler_func_ns_test.go

```diff
@@ -78,4 +80,72 @@ func TestFunctionNSCall(t *testing.T) {
 
 		So(err, ShouldNotBeNil)
 	})
+
+	Convey("Should use keywords", t, func() {
+		c := compiler.New()
+
+		keywords := []string{
```

Comment:
Is it working with `TRUE` and `FALSE`?

Reply:
Yes

Reply:
In this case, I suggest adding cases with `TRUE` and `FALSE`

Reply:
Ah, good catch. I need to "unwrap" ``BooleanLiteral`` and few other literals.